File: Advent of Code 2022/Day 1/1a.py
def main():
    input: list[str] = open("Advent of Code 2022\Day 1\input.txt").read().split("\n\n")

    max_calorie: int = 0
    for item in input:
        val: int = 0
        for value in item.split("\n"):
            val += int(value)

        max_calorie = max(val, max_calorie)

    return max_calorie


if __name__ == "__main__":
    main()

File: Advent of Code 2022/Day 1/1a.js
const fs = require("fs");

const input = fs.readFileSync("Advent of Code 2022/Day 1/input.txt", "utf-8");
const arr = input.split("\r\n\r\n");

let max_cal = 0;
for (let item = 0; item < arr.length; item++) {
	const element = arr[item].split("\r\n");
	let sum = 0;
	for (let cal = 0; cal < element.length; cal++) {
		let val = parseInt(element[cal]);
		sum += val;
	}

	max_cal = Math.max(sum, max_cal);
}

console.log(max_cal);
